modules/decklinksrc/decklinkSrcWorker.py

```diff
@@ -0,0 +1,16 @@
+from modules.decklinksrc.decklinksrc import DeckLinkSRC
+
+
+def decklinkSrcWorker(pause, exitRequest, pipelineOut):
+    print("Start decklinksrc")
+    decklinkSrc = DeckLinkSRC()
+    while True:
+        pause.acquire()
+        pause.release()
+
+        curr_frame = decklinkSrc.grab()
```

Comment:
Is the method always correct? Will there always be a valid frame returned?

Reply:
No, I added a check to make sure its valid, however we could focus on source control, and make sure that invalid data is not put in the pipeline. Issue is, we do kind of want that invalid data there for testing.